License.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Can we rename to file to LICENSE and change the licensing name to LabShare? See here for an example: https://github.com/LabShare/vision-ui/blob/master/LICENSE